pkg/gateway/multiparts/multipart.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+option go_package = "github.com/treeverse/lakefs/gateway/multiparts";
+
+import "google/protobuf/timestamp.proto";
+
+package io.treeverse.lakefs.multiparts;
+
+message MultipartUploadData {
```

Comment:
Please add documentation for the type and fields.

Reply:
We can add documentation for the message but I think the fields should be documented in the original struct